Tasks/DzmitrySidarau/Task1/Task1.py

```diff
@@ -0,0 +1,12 @@
+D = 20000
+#  D - initial deposit amount,BYN
+n = 15
+# n - deposit percentage (year),%
+T = 60
+# T - deposite term, month
+L = 12
```

Comment:
these values should be of the decimal data type, since they are money-related, -1 point for the unwise choice